CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,485 @@
+      },
+      "required": [
+          "name",
+          "social"
+      ]
+    },
+    "report": {
+      "type": "object",
+      "properties": {
+        "reportURLs": {
+          "type": "array",
+          "items": [
+              {
+                "type": "string",
+                "description": "A URL to the report of the certification/audit."
+              }
+          ]
+        },
+        "reportHash": {
+          "type": "string",
+          "description": "Hash of the report pointed by the links in reportURLs"
+        }
+      },
+      "required": [
+          "reportURLs",
+          "reportHash"
+      ]
+    },
+    "summary": {
+      "type": "string",
+      "description": "A string of the summary of the certification/audit. "
+    },
+    "disclaimer": {
+      "type": "string",
+      "description": "A string of the legal disclaimer from the Certificate Issuer"
+    },
+    "scripts": {
+      "type": "array",
+      "items": [
+        {
+            "smartContractInfo": {
+                "type": "object",
+                "properties": {
+                    "era": {
+                        "type": "string",
+                        "description": "Describe which era this script was written for."
+
+                    },
+                    "compiler": {
+                        "type": "string",
+                        "description": "Compiler used for this script."
+                    },
+                    "compilerVersion": {
+                        "type": "string",
+                        "description": "Compiler version used for this script."
+                    },
+                    "optimizer": {
+                        "type": "string",
+                        "description": "Post compilation optimizer used for this script."
+                    },
+                    "optimizerVersion": {
+                        "type": "string",
+                        "description": "Post compilation optimizer version used for this script."
+                    },
+                    "progLang": {
+                        "type": "string",
+                        "description": "Programming language used for this script"
+                    },
+                    "repository": {
+                        "type": "string",
+                        "description": "URL to the repository of the script"
+                    }
+                }
+            },
+            "fullScriptHash": {
+              "type": "string",
+              "description": "Prefix and script hash or script hash+staking key"
+            },
+            "scriptHash": {
+              "type": "string",
+              "description": "Script hash or script hash+staking key"
+            },
+            "contractAddress": {
+              "type": "string",
+              "description": "Script on-chain address"
+            },
+            "required": [
+                "scriptHash",
+                "contractAddress"
```

Comment:
I would not make contractAddress mandatory but optional because it is duplicate of scriptHash, just in bech32 form so anyone can calculate this.

Reply:
resolving: not marked as mandatory (anymore)